src/main/java/com/conveyal/taui/models/AnalysisRequest.java

```diff
@@ -83,14 +86,15 @@ public AnalysisTask populateTask (AnalysisTask task, Project project) {
             modifications = modificationsForProject(project.accessGroup, projectId, variantIndex);
         }
 
-        // No idea how long this operation takes or if it is actually necessary
+        // Generate a checksum of all the modifications
```

Comment:
If it wasn't clear to you why we were generating a CRC, it will probably confuse us again in the future. We should add a comment here saying what this is for.

Reply:
👌